include/highfive/bits/H5DataType_misc.hpp

```diff
@@ -62,6 +62,11 @@ inline bool DataType::isReference() const {
     return H5Tequal(_hid, H5T_STD_REF_OBJ) > 0;
 }
 
+//File DataType::getFile(){
+//    hid_t file_id = H5Iget_file_id(_hid);
+//    return File(file_id);
+//}
```

Comment:
Dead/commented out code should be removed. Is there a good use-case scenario where you would need to get the corresponding file for a datatype?

Reply:
I wait for help with include recursion hell. When we somehow add File definition to this file then I uncomment this (or maybe I need to uncomment it now :)